nbdime/__init__.py

```diff
@@ -18,12 +18,18 @@ def load_jupyter_server_extension(nb_server_app):
     _load_jupyter_server_extension(nb_server_app)
 
 
+_load_jupyter_server_extension = load_jupyter_server_extension
+
+
 def _jupyter_server_extension_paths():
     return [{
         "module": "nbdime"
     }]
 
 
+_jupyter_server_extension_points = _jupyter_server_extension_paths
```

Comment:
This is currently not included in `__all__`. Should it be?

Reply:
`_jupyter_server_extension_paths` wasn't in `__all__` so I didn't add the mask function `_jupyter_server_extension_points` either. If you think they should be added I can do that.